nodeJS/introduction_to_nodeJS/introduction_what_is_nodeJS.md

```diff
@@ -69,22 +70,22 @@ While you may have learned React (or any other frontend framework) before, eithe
 
 <div class="lesson-content__panel" markdown="1">
 
-1. [This short module](https://developer.mozilla.org/en-US/docs/Learn/Server-side/First_steps) on "The Server Side" from MDN is a great source for the background knowledge you need. Read through at least the first two articles posted under the 'Guides' section: Introduction to the server side and Client-Server Overview. The other two are interesting and worth reviewing, but less relevant to our immediate concerns.
-2. To gain a little more insight into the nature of Node, and to unpack the rest of the above definition, read [this article](https://medium.freecodecamp.org/what-exactly-is-node-js-ae36e97449f5).
-3. What is the Node Event Loop? You may remember about this long, but _really_ [fantastic video](https://www.youtube.com/watch?v=8aGhZQkoFbQ) from an earlier part of the course... don't skip it!
-4. Take a few minutes to go through the "Getting Started" section of the new official [Node.js website](https://nodejs.org/en/learn/getting-started/introduction-to-nodejs). Read up until, but not including, the TypeScript module.
-5. [This short video](https://www.youtube.com/watch?v=uVwtVBpw7RQ) is a great introduction as well!
+1. This [short module](https://developer.mozilla.org/en-US/docs/Learn/Server-side/First_steps) on "The Server Side" from MDN is a great source for the background knowledge you need. Read through at least the first two articles posted under the 'Guides' section: Introduction to the server side and Client-Server Overview. The other two are interesting and worth reviewing, but less relevant to our immediate concerns.
+1. To gain a little more insight into the nature of Node, and to unpack the rest of the above definition, read this [article](https://medium.freecodecamp.org/what-exactly-is-node-js-ae36e97449f5).
```

Comment:
```suggestion
1. To gain a little more insight into the nature of Node, and to unpack the rest of the above definition, read [this article about NodeJS](https://medium.freecodecamp.org/what-exactly-is-node-js-ae36e97449f5).
```

Reply:
Same reason as the previous one, we need not to include ‘this’ in the bracket. Apart from that this looks descriptive as well 👍.

Reply:
fixed